tests/lib/rules/id-length.js

```diff
@@ -126,27 +132,140 @@ ruleTester.run("id-length", rule, {
             ]
         },
         {
-            code: "var { x} = {};",
+            code: "function foo({x}) { }",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
+        {
+            code: "function foo({x: a}) { }",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
```

Comment:
> I think it would be good to add data (maybe also location?) to some of the new tests, at least to those with two or more potentially invalid names, to confirm that this reports a, not x.

I agree! it makes sense. :) thanks